Makefile

```diff
@@ -10,7 +10,7 @@ PYTHON_WITH_VERSION=python$(PYTHON_VERSION)
 SYSTEM_DEPENDENCIES=$(PYTHON_WITH_VERSION) $(PYTHON_WITH_VERSION)-dev \
 	virtualenv lsb-release pkg-config git build-essential libssl-dev tox \
 	libsnappy-dev
-OS=$(shell lsb_release -si)
+OS=$(shell lsb_release -si || echo 'not Linux')
```

Comment:
I would do a `echo N/A` because `not Linux` is misleading. Basically it's just that we don't have the `lsb_release` binary